cli/flags.go

```diff
@@ -45,112 +46,113 @@ func (v pflagValue) IsBoolFlag() bool {
+- self: for single node systems, specify --gossip=self (the
+  <address> is omitted).
 `,
-	"password": `
-        The created user's password. If provided, disables prompting. Pass '-' to provide
-	the password on standard input.
+	"key-size": wrapText(`
+Key size in bits for CA/Node/Client certificates.`),
+	"linearizable": wrapText(`
+Enables linearizable behaviour of operations on this node by making
+sure that no commit timestamp is reported back to the client until all
+other node clocks have necessarily passed it.`),
+	"dev": wrapText(`
+Runs the node as a standalone in-memory cluster and forces --insecure
+for all server and client commands. Useful for developing Cockroach
+itself.`),
+	"insecure": wrapText(`
+Run over plain HTTP. WARNING: this is strongly discouraged.`),
+	"max-offset": wrapText(`
+The maximum clock offset for the cluster. Clock offset is measured on
+all node-to-node links and if any node notices it has clock offset in
+excess of --max-offset, it will commit suicide. Setting this value too
+high may decrease transaction performance in the presence of
+contention.`),
+	"memtable-budget": wrapText(`
+Total size in bytes for memtables, shared evenly if there are multiple
+storage devices.`),
+	"metrics-frequency": wrapText(`
+Adjust the frequency at which the server records its own internal metrics.
+`),
+	"pgaddr": wrapText(`
+The host:port to bind for Postgres traffic.`),
+	"scan-interval": wrapText(`
+Adjusts the target for the duration of a single scan through a store's
+ranges. The scan is slowed as necessary to approximately achieve this
+duration.`),
+	"scan-max-idle-time": wrapText(`
+Adjusts the max idle time of the scanner. This speeds up the scanner on small
+clusters to be more responsive.`),
+	"time-until-store-dead": wrapText(`
+Adjusts the timeout for stores.  If there's been no gossiped updated
+from a store after this time, the store is considered unavailable.
+Replicas on an unavailable store will be moved to available ones.`),
+	"stores": wrapText(`
+A comma-separated list of stores, specified by a colon-separated list
+of device attributes followed by '=' and either a filepath for a
+persistent store or an integer size in bytes for an in-memory
+store. Device attributes typically include whether the store is flash
+(ssd), spinny disk (hdd), fusion-io (fio), in-memory (mem); device
+attributes might also include speeds and other specs (7200rpm,
+200kiops, etc.). For example:`) + `
+
+  --stores=hdd:7200rpm=/mnt/hda1,ssd=/mnt/ssd01,ssd=/mnt/ssd02,mem=1073741824
 `,
+	"max-results": wrapText(`
+Define the maximum number of results that will be retrieved.`),
+	"balance-mode": wrapText(`
+Determines the criteria used by nodes to make balanced allocation
+decisions. Valid options are "usage" (default) or "rangecount".`),
+	"password": wrapText(`
+The created user's password. If provided, disables prompting. Pass '-' to
+provide the password on standard input.`),
+}
+
+const wrapWidth = 79
```

Comment:
This should be less than 79 to allow room for the indentation. The default values are also getting printed at the wrong indentation level.